src/index.html

```diff
@@ -1,18 +1,21 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <meta charset="UTF-8">
     <title>Whats Cookin</title>
   </head>
   <body>
     <!-- <header>
       dddd
     </header> -->
-    <nav>
+    <nav class="top-nav" id="topNav">
       <ul>
         <li id="recipes">Recipes</li>
         <li id="favorites">Favorites</li>
         <li id="recipesToCook">Recipes to Cook</li>
+        <a href="javascript:void(0);" class="icon" onclick="myFunction()">
```

Comment:
Hey Steven, looks good, but I'm not sure the instructors want us to use onClick functionality... or any JS in html.